pkg/router/f5/f5.go

```diff
@@ -459,6 +459,35 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 }
 
 //
+// iControl REST resource helper methods.
+//
+
+// iControlResourceUriPath returns a fully qualified `escaped` resource URI path component that can be used in the iControl REST calls.
+func (f5 *f5LTM) iControlResourceUriPath(resourceName string) string {
```

Comment:
Could we rename this with something other than "UriPath"? Using "UriPath" in the name suggests the result will (1) look like a path and (2) be used as a path.  However, the result is really a path that is escaped with tildes in place of slashes, so it does not look like a path, contradicting (1); and the result is in fact interpolated by callers *into* a URI path as a single path component, contradicting (2).  Perhaps s/iControl(\S+)UriPath/\1EncodedPath/?

Same applies to `iControlVserverUriPath`, `escapeiControlUriPath`, and the various *UriPath variables.